docs/user/web_data.rst

```diff
@@ -396,37 +396,37 @@ Appendix A: GeoServer Jars
 +------------------------------------------------------------+------------+
 | GeographicLib-Java-1.44.jar                                | 31693      |
 +------------------------------------------------------------+------------+
-| geomesa-accumulo-datastore-1.2.0-SNAPSHOT.jar              | 2338358    |
+| geomesa-accumulo-datastore_2.11-$VERSION.jar               | 3059818    |
```

Comment:
does it make sense to have $VERSION here when we are listing the exact size in bytes?